# Autol LiDAR Frame.
std_msgs/Header          header
uint8 		lidar_index
AutolS56Packet[]   packets	    # array of packets
